7.5/enable-azure-service-bus.ps1

```diff
@@ -0,0 +1,111 @@
+param(
```

Comment:
When allowing here to reference the service bus the same way, as it is during creation, we could drop the connection string by retrieving it freshly, which would reduce user errors.

Reply:
Another good idea. Of course, for both changes you can expect that I must also change the description for the user.

Reply:
Following occurred to me. If we remove connection string as parameter, then we need all the other parameters from the other script. I can imagine that our users either know what they are doing (they create service bus alone and know what is connection string) or they have no idea (they use script for creation, which gives them connection string). What do you think? I think now that we should keep connection string.